plugins/pyembed/pyembed.cpp

```diff
@@ -156,6 +156,8 @@ class PythonThreadContext
     {
         PyEval_RestoreThread(threadState);
         script.clear();
+        module.clear();
```

Comment:
This change was for clarity - does not affect the behaviour as they would already have been cleared during the time that we own the GIL, but that fact is clearer by making them explicit